apps/site-landing/src/containers/HomePage/underwater/styles.module.less

```diff
@@ -25,7 +26,7 @@
 
   .body {
     margin-top    : -8px;
-    padding-top   : .scalable(20, 150)[@result];
+    padding-top   : .scalable-clamped-bp(50, 124, @mobileWidth, @midWidth, px)[@result];
```

Comment:
I do worry that putting this much math into my CSS will end up killing page performance.